packages/types/src/config.ts

```diff
@@ -136,6 +136,12 @@ export interface SlidevConfig {
    * @default 'dev'
    */
   record: boolean | 'dev' | 'build'
+  /**
+   * Options for presenter mode
+   *
+   * @default {public:true}
```

Comment:
The default does not match with types I think

Reply:
Thanks for noticing, I forgot to update here